testing/procs/APKBUILD

```diff
@@ -5,12 +5,18 @@ pkgver=0.8.9
 pkgrel=0
 pkgdesc="A modern replacement for ps written in Rust"
 url="https://github.com/dalance/procs"
-arch="x86_64 armhf armv7" # limited by rust/cargo
+arch="x86_64 armv7 armhf !aarch64 x86 ppc64le" # limited by rust/cargo (aarch64 due to MAP_32BIT in nix package)
 license="MIT"
 options="net"
 makedepends="cargo"
 source="$pkgname-$pkgver.tar.gz::https://github.com/dalance/procs/archive/v$pkgver.tar.gz"
 
+case "$CARCH" in
+	x86)
+		export CFLAGS="$CFLAGS -fno-stack-protector"
```

Comment:
Might be worth putting a comment on this

Reply:
Done